examples/meta/generator/targets/scala.json

```diff
@@ -0,0 +1,99 @@
+{
```

Comment:
As said before, this file can go. We just want to execute the already present java files